modules/product/index.js

```diff
@@ -16,6 +19,16 @@ module.exports = {
           }
         }
       }
+    },
+    groups: {
```

Comment:
Am I supposed to see tabs in the modal when creating a new product?

Reply:
Yes, you should if you are using the latest version of the `3.0` branch in core.

Reply:
It is ok now. Thanks